Automattic/jetpack
# SEO Tools: Fix Custom SEO Titles unique key prop warnings

#### Changes proposed in this Pull Request:

* This PR adds and moves key props within two Custom SEO Titles components. This fixes two console errors warning that each child in a list should have a unique "key" prop.

#### Other information:


#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* To replicate (without applying this PR):
  * Make sure script debug is enabled: `define( 'SCRIPT_DEBUG', true );`  in wp-config.php.
  * From wp-admin go to Jetpack → Settings → Traffic and activate the “Customize your SEO settings” option under the “Search engine optimization” section. Then expand the section 'Expand to customize the page title structures of your site'.
  * In the console you should see a couple of errors saying 'Warning: Each child in a list should have a unique "key" prop.' - related to `CustomSeoTitleInput` and `CustomSeoTitles`.
* Apply this PR, then follow the same steps as above - you shouldn't see any console errors.
* Customize the title structure for a post / page - then view the page source that post / page to make sure the custom page / post title is showing under the Jetpack OG tags - just to confirm everything is still working properly.
* If you have React DevTools Chrome extension installed (if using Chrome), you can also open up the 'Components' tab within the Developer Tools. You should be able to see the newly added key names linked to the `CustomSeoTitleInput` and `SEOTokenButton` components.
